index.html

```diff
@@ -93,6 +93,34 @@ <h2>Publications</h2>
         </header>
         <ul class="10u -1u">
 
+          <li>
+        <a href="publications/2018/hagedorn18Stencils.pdf">
+High Performance Stencil Code Generation with Lift
+        </a><br>
+        Bastian Hagedorn, Larisa Stoltzfus, Michel Steuwer, Sergei Gorlatch, Christophe Dubach
+        <a href="http://cgo.org/cgo2018/">
+International Symposium on Code Generationand Optimization (CGO) 2018
+        </a><br>		
+        <a href="publications/2017/hagedorn18Stencils.pdf" style="margin-right:1em">

```

Comment:
Wrong year in the path here.

Reply:
the path will be introduced here https://github.com/lift-project/publications/pull/2

Reply:
Yeah, but it is 2018/hagedorn18Stencils.pdf, not 2017/hagedorn18Stencils.pdf